zwave_js_server/const/command_class/thermostat.py

```diff
@@ -75,6 +89,14 @@ class ThermostatSetpointType(IntEnum):
     AWAY_COOLING = 14
     FULL_POWER = 15
 
+    @classmethod
+    def _missing_(cls: type, value: object) -> ThermostatSetpointType:
+        """Set default enum member if an unknown value is provided."""
+        # Handle reserved values
+        if isinstance(value, int) and 3 <= value <= 6:
```

Comment:
What has reserved 3-6?

Reply:
It's reserved in the spec, I guess it would be device specific? To kpine's point below, this solution doesn't work, will close and open a new one when ready